__tests__/src/adapters/nextjs-sparrow/NextJsUrlManager.test.ts

```diff
@@ -3,10 +3,65 @@
  */
 import { NextJsUrlManager } from "../../../../src/adapters/nextjs-sparrow/NextJsUrlManager";
 
+/* eslint-disable jest/valid-title */
+/* eslint-disable @typescript-eslint/unbound-method */
+
 describe(NextJsUrlManager.nodeNameUpdate, () => {
   it("should return correct endpoint url", () => {
     expect(
       NextJsUrlManager.nodeNameUpdate("gfoo", "sbar")
     ).toMatchInlineSnapshot(`"/api/gateway/gfoo/node/sbar/config"`);
   });
 });
+
+describe(NextJsUrlManager.presentNotifications, () => {
+  it("returns the base API endpoint when no ids are present", () => {
+    expect(NextJsUrlManager.presentNotifications()).toMatchInlineSnapshot(
```

Comment:
TIL `toMatchInlineSnapshot` - I didn't know that was a method in Jest. Very handy